src/main/java/org/apache/commons/compress/compressors/gzip/GzipCompressorOutputStream.java

```diff
@@ -185,14 +196,26 @@ private void writeHeader(final GzipParameters parameters) throws IOException {
             buffer.put((byte) 0);
         }
         buffer.put((byte) parameters.getOperatingSystem());
+
```

Comment:
Remove extra whitespace, if you need to point something out, use an `// inline comment`.